src/unwinder/i386/unwinder_i386.cpp

```diff
@@ -97,6 +97,10 @@ OOPStackUnwinderX86::VirtualUnwind(
 
     ContextRecord->ContextFlags |= CONTEXT_UNWOUND_TO_CALL;
 
+#define CALLER_SAVED_REGISTER(reg) if (rd.pCurrentContextPointers->reg) ContextRecord->reg = *rd.pCurrentContextPointers->reg;
```

Comment:
This is not needed. The UnwindStackFrame never sets context pointers of the caller saved registers.